sensor/kubernetes/listener/resources/rbac/binding_fetcher_test.go

```diff
@@ -0,0 +1,48 @@
+package rbac
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func Test_FetchBindingRemovesLastAppliedConfig(t *testing.T) {
```

Comment:
I would maybe add another case with a `Role` just to cover both paths in `generateDependentEvent`.